packages/hops/lib/runtime.d.ts

```diff
@@ -0,0 +1,83 @@
+import { BrowserRouterProps } from 'react-router-dom';
+
+declare module 'hops' {
+  type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+  export class Miss extends React.Component<any, any> {}
+
+  export interface StatusProps {
+    code?: number;
```

Comment:
`code` is a required prop & also `staticContext` seems to be missing.

Reply:
I changed code to be required. `staticContext` is injected and therefore not required to pass in from the outside, that's why I left it out of this definition.